### PR TITLE
juniper_show_route_exact_active_path

### PR DESCRIPTION
**Pull request by rvisnu**
##### ISSUE TYPE

 - New Template Pull Request
 - Bugfix Pull Request
 - Additional Testing
 - Docs Pull Request

##### SUMMARY

```

```

### CODE REVIEW

File: templates/juniper_junos_show_route_exact_active-path.template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Value SUBNET (\S+\/\S+)
+Value TIME_IN_TABLE (\S+\s+\S+)
```
Comment (sidtyagi) [inline templates/juniper_junos_show_route_exact_active-path.template]: please check if the naming convention is right (are _ allowed ?)

File: templates/index [hunk 2]
```diff
@@ -241,6 +241,7 @@ hp_procurve_show_system.template, .*, hp_procurve, sh[[ow]] syst[[em]]
 hp_procurve_show_vlans.template, .*, hp_procurve, sh[[ow]] vl[[ans]]
 hp_procurve_show_arp.template, .*, hp_procurve, sh[[ow]] ar[[p]]
 
+juniper_junos_show_route_exact_active-path.template, .*, juniper_junos|juniper, sh[[ow]] rou[[te]] exa[[ct]] (\S+) act[[ive-path]]
```
Comment (sidtyagi) [inline templates/index]: hyphen is allowed ?

File: tests/juniper_junos/show_route_exact_active_path/juniper_junos_show_route_exact_active-path_v6.parsed [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (sidtyagi) [inline tests/juniper_junos/show_route_exact_active_path/juniper_junos_show_route_exact_active-path_v6.parsed]: please add raw output for v6 a well

File: templates/index (hunk 2 above)
Comment (rvisnu) [reply templates/index]: We can see hyphen usage in other templates.

File: templates/juniper_junos_show_route_exact_active-path.template (hunk 1 above)
Comment (rvisnu) [reply templates/juniper_junos_show_route_exact_active-path.template]: We have _ in other scenarios. I will change if it is not standard

File: tests/juniper_junos/show_route_exact_active_path/juniper_junos_show_route_exact_active-path_v4.raw [hunk 4]
```diff
@@ -0,0 +1,11 @@
+show route exact 1.2.3.0/24 active-path
```
Comment (Sandeepsr) [inline tests/juniper_junos/show_route_exact_active_path/juniper_junos_show_route_exact_active-path_v4.raw]: Can you please check this for other SKU  and roles we have in Juiper,

File: tests/juniper_junos/show_route_exact_active_path/juniper_junos_show_route_exact_active-path_v6.parsed (hunk 3 above)
Comment (rvisnu) [reply tests/juniper_junos/show_route_exact_active_path/juniper_junos_show_route_exact_active-path_v6.parsed]: added raw output for v6

File: tests/juniper_junos/show_route_exact_active_path/juniper_junos_show_route_exact_active-path_v4.raw (hunk 4 above)
Comment (rvisnu) [reply tests/juniper_junos/show_route_exact_active_path/juniper_junos_show_route_exact_active-path_v4.raw]: Working for BGP on multiple skus

**Pull request merged**